tuf/formats.py

```diff
@@ -356,6 +348,7 @@
 TIMESTAMP_SCHEMA = SCHEMA.Object(
   object_name = 'TIMESTAMP_SCHEMA',
   _type = SCHEMA.String('timestamp'),
+  spec_version = SPECIFICATION_VERSION_SCHEMA,
```

Comment:
Does this require new tests?

Reply:
This in itself, no.  The spec version was simply missing in the schema for Timestamp.  It was already being written in metadata despite the fact that it was missing in the schema, because this schema was not really used to build the metadata before.  Now the schema is used to build metadata.

I want to add more validation that uses these schemas in updater, and to add some additional testing for formats.py related to the schemas more generally, but that's not really related to this change, and it'll have to wait.